### ISSUE TITLE
Show connection name and support particular size import when import data between two connections

### ISSUE THREAD

**Issue by Bitros** [dbeaver/dbeaver#19080]
Hi all,

I have a good suggestion for data import between 2 DBeaver connections which will improve user experience.

Recently, I want to import some production data into development database and found that DBeaver already support importing data between 2 two tables in separated database. However, the title does not display the connection name. It may confused and could make terrible changes carelessly.

![image](https://user-images.githubusercontent.com/9913942/219328413-91ae1ae3-276a-4d4a-9c5f-753957322aa3.png)

In this importing case, I just want to do some test and do not need to import all the data. But I did not find any settings support this (If I missed, thanks for sharing). It will be great if we can control the size of importing data. 

In short

**1. Display connection name when import data
2. Let user control the size of importing data**

**Comment by E1izabeth** [comment]
Thank you for the idea.
